packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -5584,7 +5584,12 @@ class EditableTextState extends State<EditableText>
                         excludeFromSemantics: true,
                         axisDirection: _isMultiline ? AxisDirection.down : AxisDirection.right,
                         controller: _scrollController,
-                        physics: widget.scrollPhysics,
+                        // On iOS a single-line TextField should not scroll.
+                        physics:
+                            widget.scrollPhysics ??
```

Comment:
I'm glad this is overridable if the developer still wants to enabled scrolling :+1: